.github/workflows/main.yml

```diff
@@ -22,109 +22,109 @@ jobs:
         # https://hexdocs.pm/elixir/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp
         include:
           # Newest supported Elixir/Erlang pair.
-          - elixir: '1.18'
-            otp: '27.2'
+          - elixir: "1.18"
+            otp: "27.2"
             lint: true
             dialyzer: true
             os: ubuntu-latest
 
           # One version before the last supported one.
-          - elixir: '1.17'
-            otp: '26.2'
+          - elixir: "1.17"
+            otp: "26.2"
             os: ubuntu-22.04
 
           # Oldest supported Elixir/Erlang pair.
-          - elixir: '1.13.4-otp-22'
-            otp: '22.3.4'
-            os: ubuntu-20.04
+          - elixir: "1.13"
+            otp: "25.3"
```

Comment:
We need to leave a comment here saying that this is the lowest *tested* OTP version but technically we support back to OTP 22.